web/themes/custom/pineapple/sass/base/forms/_filters-form.scss

```diff
@@ -0,0 +1,59 @@
+.view-recipes {
```

Comment:
This file should be named after the specific form, "filters form" is too general especially considering you may have other exposed forms in views

Reply:
That file is going to contain all other filter forms. It doesn't have anything to do with views. I just used that selector to target this specific form.

Reply:
If you have other filters they should be in separate files, code especially css easily gets messy because people tend to include it all in one file. If you have a different filter form that has things which are similar to this form then by all means create one main filter form file which has code from both forms and include it in there but the general rule is to keep a good level of separation

Reply:
I understand this rule, we have discussed it in the past, however I do not expect filter forms to have styling which varies much. I will likely have one file for all filter forms in the project.

Reply:
Okay if that case your best off removing the "view-recipes" class from the file because right now the filter is only being stlyed for the recipe view

Reply:
Agreed, I have done that and replaced it with a form class. Thank you.